client-core/src/wallet/default_wallet_client.rs

```diff
@@ -293,10 +295,12 @@ where
                 self.key_service
                     .add_wallet_private_key(name, &private_key, &enckey)?;
 
-                self.wallet_service.create(name, &enckey, public_key)?;
+                self.wallet_service
+                    .create(name, &enckey, public_key, wallet_kind)?;
 
                 Ok((enckey, Some(mnemonic)))
             }
+            WalletKind::HW => todo!("find hardware wallet as private key, and create a new wallet"),
```

Comment:
you can still create the "HW wallet" -- the only thing will be that the operations requiring `sign` or `public_key` will fail?

Reply:
Yes I can, it will give an unreachable warning when compile if I use the `hw_wallet.public_key()`, because the `hw_wallet` comes from a `todo!()`.